package.json

```diff
@@ -16,18 +16,18 @@
   },
   "main": "index.js",
   "scripts": {
-    "build": "craco build && yarn run generate-version-file",
+    "build": "react-scripts build && yarn run generate-version-file",
     "clean": "./scripts/clean-files",
     "eslint-check": "eslint src/",
     "generate-release-tarball": "yarn run build && ./scripts/generate-release-tarball",
     "generate-version-file": "./scripts/generate-version-file",
     "lint": "yarn run eslint-check && yarn run prettier-check && yarn run stylelint-check",
     "prettier-check": "prettier --check 'src/**/*'",
     "serve": "yarn run start",
-    "start": "craco start",
+    "start": "react-scripts start",
     "stylelint-check": "stylelint 'src/**/*.scss'",
     "stylelint-fix": "stylelint --fix 'src/**/*.scss'",
-    "test": "craco test"
+    "test": "react-scripts test --transformIgnorePatterns 'node_modules/(?!@canonical)/jujulib'"
```

Comment:
While the app runs and builds properly, the tests fail if we do not babelize jujulib to remove the `import` statements. This has the unfortunate side effect of making circleci run out of memory so I'll have to dig into this in the morning.